modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -10,6 +10,15 @@ import {xSmallIcon} from '@workday/canvas-system-icons-web';
 import {TextInputProps} from '@workday/canvas-kit-react-text-input';
 import uuid from 'uuid/v4';
 
+export interface ComboBoxMenuItemProps extends MenuItemProps {
+  shortText?: string;
```

Comment:
Can you give a little more insight on how this is meant to be used? In this case, the picture doesn't really say 1000 words 😝

Reply:
So let's say a autocomplete item has a bunch of help text:

```
<MenuItem>
  John Smith
  San Francisco | Programmer | Loves Dogs
</MenuItem>
```

But when you click on that item you only want the main text `John Smith` to fill in input on click.

Does that help?

Reply:
Yes it does, thanks. Not sure `shortText` is the best name for this IMO. What do you think about just calling it `value`? This aligns with how a radio behaves for example.

Reply:
The only problem with value is it is a real attribute and isn't usually associated with `<li>`s